source/fuzz/fuzzer.cpp

```diff
@@ -261,6 +261,8 @@ void Fuzzer::MaybeAddFinalPass(std::vector<std::unique_ptr<FuzzerPass>>* passes,
 }
 
 bool Fuzzer::ApplyPassAndCheckValidity(FuzzerPass* pass) const {
+  static uint32_t counter = 0;
```

Comment:
Not sure why we need this.

Reply:
Oh dear, well spotted! I was having trouble debugging so introduced this in order to do a counting breakpoint, and then forgot to remove it!